nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -91,6 +91,10 @@
 
 ## Backward Incompatibilities {#sec-release-23.11-incompatibilities}
 
+- `multi-user.target` no longer depends on `network-online.target`.
+  This will potentially break services that assumed this was the case in the past.
+  This was changed for consistency with other distributions as well as improved boot times.
```

Comment:
Conflicts with other changes, pls rebase.

Reply:
bluh. will do this once more is done, i rebased quite recently onto nixos-unstable to try to get hydra to have more stuff in the cache so that we can get it to tell us about failed tests.

Reply:
You should stuff your changes in a random position to avoid conflicts; the entries will be reordered anyway later, during the release process.